scripts/deploy-olm.sh

```diff
@@ -69,8 +70,15 @@ then
   NAMESPACE=default
 fi
 
+# Get the namespace where the catalog is
+if scripts/is-openshift.sh
```

Comment:
Instead of hardcoding `scripts`, can we define SCRIPT_DIR and use that.  That way you can use the script from any directory.  We set SCRIPT_DIR in other scripts.

```
SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```